client-v2/src/actions/FrontsConfig.js

```diff
@@ -28,6 +45,79 @@ function errorReceivingConfig(error: string): Action {
   };
 }
 
+function getFrontCollection(collectionId: string) {
+  return (dispatch: Dispatch, getState: () => State) =>
+    getCollection(collectionId)
+      .then((res: Object) => {
+        const collectionConfig = getCollectionConfig(getState(), collectionId);
+        const collectionWithNestedArticles = combineCollectionWithConfig(
+          collectionConfig,
+          res
+        );
+        const collectionWithDraftArticles = {
+          ...collectionWithNestedArticles,
+          draft: populateDraftArticles(collectionWithNestedArticles)
+        };
+        const {
+          collection,
+          articleFragments
+        } = normaliseCollectionWithNestedArticles(collectionWithDraftArticles);
+
+        dispatch(
+          batchActions([
+            collectionReceived(collection),
+            articleFragmentsReceived(articleFragments)
+          ])
+        );
+        return collectionWithDraftArticles.live
+          ? [
+              // We use a set to dedupe our article ids
+              ...new Set([
```

Comment:
Pretty sure this will be polyfilled automatically by 'useBuiltIns' in babel but might be worth a sanity check!

Reply:
Good point - our supported browsers as indicated in babel.config.js are fine with this, but I'll add missing imports for babel-polyfill, so future use of unsupported ESx-y things should be polyfilled as needed.